TeamCode/src/main/java/org/firstinspires/ftc/teamcode/FourWheelsDriveBot.java

```diff
@@ -178,5 +198,22 @@ public void driveStraightByDistance(int direction, double distance, double maxPo
                 rightFront.getCurrentPosition(),
                 leftRear.getCurrentPosition(),
                 rightRear.getCurrentPosition()));
+    }
+    public void pickupSkyStone(){
+        opMode.telemetry.log().add("pickupSkyStone()");
+        int i = 1;
+        while (this.opMode.opModeIsActive() && i < 2) {
+            opMode.telemetry.log().add(String.format("pickupSkystone loop %d, %.2f, %.2f", i, servoArm.getPosition(), servoPinch.getPosition()));
+
+            servoArm.setPosition(servoArm.getPosition() - 0.5);
+            servoPinch.setPosition(servoPinch.getPosition() - 0.3);
+            opMode.sleep(CYCLE_MS);
+            opMode.telemetry.log().add(String.format("pickupSkystone end %d, %.2f, %.2f", i, servoArm.getPosition(), servoPinch.getPosition()));
+            opMode.idle();
```

Comment:
This line is great, but can be better in another way. 

```suggestion
            opMode.idle();
```